java/change-notes/2021-08-10-cleartext-storage-sharedprefs-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The query "Cleartext storage of sensitive information using `SharedPreferences` on Android" (`java/android/cleartext-storage-shared-prefs`) has been promoted from experimental to the main query pack. Its results will now appear by default. This query was originally [submitted as an experimental query by @luchua-bc](https://github.com/github/codeql/pull/4675).
```

Comment:
Change note needs to be moved and updated to the new format.

Reply:
Done!